### PR TITLE
TOC, tl;dr and additional discussion

### PR DESCRIPTION
**Pull request by b1tr0t**
TOC, tl;dr and additional discussion

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -1,5 +1,66 @@
 # Page Embedded Permission Control (PEPC)
 
+## tl;dr
+
+We propose a semantic permission element with styling constraints that ensures a
+very high level of confidence concerning *user intent* to make a permission
+decision. We believe this solves user problems related to accessibility,
+context, regret and more. The permission element is more accessible and more
```
Comment (tomayac) [inline explainer.md]:
```suggestion
context, regret, and more. The permission element is more accessible and more
```

File: explainer.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+secure than the current current permission journeys as the semantic element is
```
Comment (tomayac) [inline explainer.md]:
```suggestion
secure than the current permission flows as the semantic element is
```

File: explainer.md [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+by default more screen reader friendly than JavaScript and full page modal
```
Comment (tomayac) [inline explainer.md]: > more screen reader friendly than JavaScript

[citation needed] I would also rephrase this somehow and say it's about JS-initiated permission flows. No idea to be honest how the actual permission prompt gets presented to assistive technology, but the JS-initiated flow should just be exposed as, typically, a button, which works great with AT.

Comment (tomayac) [inline explainer.md]:
```suggestion
by default more screen reader friendly than JavaScript. Full page modal
```

File: explainer.md [hunk 4]
```diff
@@ -1,5 +1,66 @@
...
+confirmation UI reduces risk of change blindness and makes it more difficult for
```
Comment (tomayac) [inline explainer.md]:
```suggestion
confirmation UI reduces the risk of change blindness and makes it more difficult for
```

File: explainer.md [hunk 5]
```diff
@@ -1,5 +1,66 @@
...
+sites to manipulate the user's decision making by obscuring site content during
+the critical decision moment of the journey. A semantic element and modal UI are
+connected in the same proposal as accurately capturing user intent is essential
+to reducing the modal's potential for user annoyance. While we believe
+improvements to existing capability based permission journeys can and should be
```
Comment (tomayac) [inline explainer.md]:
```suggestion
improvements to existing capability based permission flows can and should be
```

File: explainer.md [hunk 6]
```diff
@@ -1,5 +1,66 @@
...
+explored, we believe there to have limited headroom for optimization, and that
```
Comment (tomayac) [inline explainer.md]:
```suggestion
explored, we believe there to be limited headroom for optimization, and that
```

Comment (tomayac) [reply explainer.md]: I'd add what optimizations could be explored, for example, moving from a line-of-death prompt on desktop or modal on mobile to a more locally-aware prompt that shows close to where the user gesture happened.

**Review by tomayac — COMMENTED** [review]
I didn't get all the way through, but maybe already some useful comments.

**Comment by b1tr0t** [comment]
Merged suggestions

File: explainer.md (hunk 3 above)
Comment (b1tr0t) [reply explainer.md]: I'll link to the accessibility discussion later -- it's not just screen readers, but also magnification users as PEPC modal makes it impossible for the user to miss that they are being prompted for a response.  (Whereas currently the user's viewport might exclude the permission request dialog.)

**Pull request closed without merging**